packages/analyticsdx-template-lint/src/schemas/variables-schema.json

```diff
@@ -72,6 +78,28 @@
             "SobjectType",
             "SobjectFieldType",
             "StringType"
+          ],
+          "enumDescriptions": [
+            "References a list of values, controlled by the `itemsType` variable type.",
+            "References true or false value.",
+            "References connectors defined in the org.",
+            "References calculated insights defined in the org. Only available in Summer '23 release.",
```

Comment:
These will show in the hover text on the `type` in `variableType` for each type. I did this since I wanted some text in there about not being available until 244 release. These are paraphrased from what's in the template dev guide's variables.json section.